packages/image-block/src/components/Image.tsx

```diff
@@ -47,7 +49,7 @@ export const ImageComponent = ({
             loading="lazy"
             src={src}
             alt={blockSettings.altText || undefined}
-            style={imageStyle}
+            style={{ maxWidth: image.width }}
```

Comment:
After looking into this a bit I wonder if it would be a good idea to add 'height' and 'width' attributes to the image. They seem to only be used to maintain the aspect ratio but it helps reduce layout shifting

Reply:
I added the height and width attributes, I like the idea. My only concern is that it might need some refactoring once the page container is not fixed to 800px but for now it seems to be an improvement.

Reply:
I actually meant directly setting them on the img element, so it shouldn't cause us any trouble

```
<img width={image.width} height={image.height}/>
```

Reply:
sure, changed to that